src/game/game_state.c

```diff
@@ -618,6 +624,7 @@ void game_state_call_move(game_state *gs) {
 }
 
 void game_state_tick_controllers(game_state *gs) {
+    controller_tick(gs->menu_ctrl, gs->int_tick, &gs->menu_ctrl->extra_events);
```

Comment:
So the menu is only statically ticked?

Reply:
Yes, that was my intention.
Dynamic ticks speed up and slow down during hitstun and other gameplay things.